DuckDuckGo/SettingsGeneralView.swift

```diff
@@ -37,6 +37,13 @@ struct SettingsGeneralView: View {
             NavigationLink(destination: WidgetEducationView()) {
                 SettingsCellView(label: UserText.settingsAddWidget)
             }
+            
+            if #available(iOS 15.0, *) {
+                NavigationLink(destination: SubscriptionPIRView()) {
+                    SettingsCellView(label: UserText.settingsAddWidget)
+                }
+            }
+
```

Comment:
This is probably some leftover from testing.

Reply:
OOF, yea.  😅,. thanks!